osu.Game/Beatmaps/Beatmap.cs

```diff
@@ -81,9 +81,12 @@ public Beatmap()
 
         public double GetMostCommonBeatLength()
         {
+            if (!HitObjects.Any())
+                return ControlPointInfo.TimingPoints.LastOrDefault()?.Time ?? 0;
```

Comment:
This is not a correct value to return from this method. The returned value should be a beat length. This is the absolute time of the last timing point.

The value cannot be early-returned, it must be assigned to `lastTime` if the beatmap has no objects.

Reply:
Yeah, apologies, I made this change in a hurry and didn't get to self-reviewing.